cmd/readOnlyLib/config.go

```diff
@@ -0,0 +1,71 @@
+package main
```

Comment:
Other folders are named like `foo-bar`, not `fooBar`. So maybe `cmd/read-only-lib`?